.github/pull_request_template.md

```diff
@@ -2,18 +2,28 @@
 
 ## Related issue number (if any)
 
-## Loom demo (if any)
-
-## Checklist before requesting a review
-- [ ] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
-- [ ] I have performed a self-review of my code.
-- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
-- [ ] If this is a new feature, I have added unit tests and integration tests.
-- [ ] I have run the integration tests locally and they are passing.
-- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
-- [ ] All features on the UI continue to work correctly.
-- [ ] Added one of the following CI labels:
-    - `run_integration_test`: Runs integration tests
-    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)
-
+## Test Plans
+Reviewer note: by approving the PR, reviewer agree that the test plans are solid enough to merge, with risk low enough for daily tests / release eng to take over.
+### Tests specific to this PR
+Include tests specific to this PR's context here:
+### Tests based on components touched by this PR
+Check the components touched by this PR and ensures all items under that component are checked. If certain item is not checked, explicitly state why and agree with PR reviewers.
+- [ ] This PR involves SDK changes
```

Comment:
Small nit:
"This PR involves SDK changes. I have:
  - included an integration test that covers my change
  - added the 'run_integration_test' label to this PR
  - (if including user-facing APIs) verified that this change is backwards compatible 
  
  etc.